io/disk/Avago_storage_adapter/avago9361.py

```diff
@@ -76,7 +76,16 @@ def test_set_on_off(self):
         """
         adjust_on_off = ['restorehotspare', 'autorebuild', 'copyback', 'eghs',
                          'alarm', 'foreignautoimport', 'maintainpdfailhistory',
-                         'ocr']
+                         'ocr', 'immediateio', 'largeQD',
+                         'driveactivityled', 'flushwriteverify',
+                         'limitMaxRateSATA', 'supportssdpatrolread',
```

Comment:
These are basically MFC(megaraid feature controller) values and its better to test all these values anyway. So I have given in the script instead of in yaml.